src/platform/windows/init.rs

```diff
@@ -24,9 +25,9 @@ use kernel32;
 use dwmapi;
 use user32;
 
-pub fn new_window(window: &WindowAttributes) -> Result<Window, CreationError> {
+pub fn new_window(window: &WindowAttributes, pl_attribs: &PlatformSpecificWindowBuilderAttributes) -> Result<Window, CreationError> {
     let window = window.clone();
-
+    let attribs = pl_attribs.clone();
```

Comment:
Passing by reference causes a lifetime error that I don't know how to fix. Since we are cloning WindowAttributes I did the same.